truffenyi-commune-quest.lic

```diff
@@ -0,0 +1,69 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#truffenyi-commune-quest
+=end
+
+custom_require.call(%w(common))
+
+echo "This script does the actions for the truffenyi commune Cleric quest. It should be started after drinking the vial two times."
+fput("stow #{checkright || checkleft}") if DRC.left_hand || DRC.right_hand
+
+while line = get
+  if line =~ /Your vision fades and you suddenly see yourself working in front of a glowing forge/
+    fput('pray Divyaush')
+  end
+  if line =~ /Your vision fades and you see yourself toiling in a dusty field. Each withered crop/
+    fput('pray Berengaria')
+  end
+  if line =~ /Your vision fades and you see yourself huddled in front of a fire in an icy cavern/
+    fput('pray Kunio')
+  end
+  if line =~ /Your vision fades and you see yourself surrounded by occupied cots. The air is still/
+    fput("pray Peri'el")
+  end
+  if line =~ /Your vision fades and you see yourself alone on a raft, calm seas in all directions./
+    fput('pray Lemicus')
+  end
+  if line =~ /Your vision fades and you see yourself as a young child sitting in the corner. You watch your parents/
+    fput('pray Albreda')
+  end
+  if line =~ /Your vision fades and you find yourself travelling the desert/
+    fput('pray Murrula')
+  end
+  if line =~ /Your vision fades and you see yourself tired and sore after a long day of harvesting crops/
+    fput('pray Rutilor')
+  end
+  if line =~ /Your vision fades and you see yourself sitting on a bar stool/
+    fput('pray Saemaus')
+  end
+  if line =~ /Your vision fades and you see yourself walking through one of your grain fields/
+    fput('pray Asketi')
+  end
+  if line =~ /Your vision fades and you see yourself sitting amongst a group gathered at an outdoor wedding/
+    fput("pray Be'ort")
+  end
+  if line =~ /Your vision fades and you see yourself sitting on a grassy hilltop/
+    fput('pray Dergati')
+  end
+  if line =~ /In your vision the waters pull away from the shore, gathering upward in a massive wall/
+    fput('pray Drogor')
+  end
+  if line =~ /Your vision fades and you see yourself seated in the front row of a concert hall/
+    fput('pray Idon')
+  end
+  if line =~ /Your vision fades and you see yourself entertaining a neighboring farmer at your house/
+    fput('pray Kerenhappuch')
+  end
+  if line =~ /Your vision fades and you see yourself battling a small peccary/
+    fput('pray Trothfang')
+  end
+  if line =~ /Your vision fades and you see yourself standing in the snow peering into the window of a rival/
+    fput('pray Zachriedek')
+  end
+  if line =~ /Your stomach grumbles and you realize that you're holding a/
+    fput("drop #{checkright || checkleft}") if DRC.left_hand || DRC.right_hand
+  end
+  if line =~ /you have my attention, though really you are never far from my sight/
+    echo('All done!')
+    exit
+  end
+end
```

Comment:
Does this incur roundtime? You could throw in a `waitrt?` here to block other scripts who are waiting for it to finish.

Reply:
there is no RT for any of the actions, but its expected that this is run afk basically.. I used fputs because there was no way to know the success messaging for every one from one quest run, should I add waitrt?'s before each fput maybe just in case someone is running another script?

Reply:
That might be the safest option

Or maybe a `waitrt?` at the top before parsing a line?

Reply:
Like this?
```while line = get
  waitrt?
  if line =~ /Your vision fades and you suddenly see yourself working in front of a glowing forge/
    fput('pray Divyaush')``` I don't want to get this wrong, I only tested what I submitted and am worried about changing it because I can never test it again :)

Reply:
actually nevermind, thats easy to test by doing something giving rt then doing a say Your vision fades and you suddenly see yourself working in front of a glowing forge

Reply:
done and tested, thanks :)